config/config.go

```diff
@@ -604,3 +609,8 @@ func (c *RemoteReadConfig) UnmarshalYAML(unmarshal func(interface{}) error) erro
 	// Thus we just do its validation here.
 	return c.HTTPClientConfig.Validate()
 }
+
+// IsZero is used to display false boolean values in the prometheus config page, as true is the default.
```

Comment:
What's this for?

Reply:
This used so that when you look the configuration in the webui you only see the honor_timestamps: false. Without this you see only the honor_timestamps: true

Reply:
You can remove the omitempty to show a false.

Reply:
I want to hide honor_timestamps: true.

My preference would have been to hide it, as we do for honor_label, because it is not globally overridable and there is a default value.

I would also like to hide 

  metrics_path: /metrics
  scheme: http

in a future pr.

It will improve readability of the config page.

Reply:
This would be inconsistent with how we render things elsewhere, and the YAML library we're using plans to add the ability to keep comments which will make this all much nicer in future.

Reply:
ack, thanks